### PR TITLE
feat(apple): Add categories to config

### PR DESCRIPTION
**Pull request by philipphofmann**
Opened because https://github.com/getsentry/sentry-docs/pull/2848 and https://github.com/getsentry/sentry-docs/pull/2840

### CODE REVIEW

File: src/platforms/apple/config.yml [hunk 1]
```diff
@@ -4,3 +4,7 @@ caseStyle: canonical
 supportLevel: production
 aliases:
   - cocoa
+categories:
+  - mobile
+  - desktop
+  - server
```
Comment (marandaneto) [inline src/platforms/apple/config.yml:10]: does sentry cocoa support server? TIL

**Review by PeloWriter — APPROVED** [review]

File: src/platforms/apple/config.yml (hunk 1 above)
Comment (bruno-garcia) [inline src/platforms/apple/config.yml:10]: Wouldn't really work on server due to single hub

Comment (bruno-garcia) [inline src/platforms/apple/config.yml:10]:
```suggestion
```

File: src/platforms/apple/config.yml [hunk 2]
```diff
@@ -4,3 +4,6 @@ caseStyle: canonical
 supportLevel: production
 aliases:
   - cocoa
+categories:
+  - mobile
+  - desktop
```
Comment (bruno-garcia) [inline src/platforms/apple/config.yml:9]: For watchOS, tvOS we could add wearable and .. iot?

```suggestion
  - desktop
  - wearable
```

Comment (marandaneto) [reply src/platforms/apple/config.yml:9]: it'd be an invalid value though, I'd stick to mobile desktop/mobile only

https://github.com/getsentry/sentry-docs/blob/dd7208a0cd9d82fa742a6847fcb1f2c8d4d423ed/src/gatsby/createSchemaCustomization/platformSchema.ts#L16-L22

Comment (bruno-garcia) [reply src/platforms/apple/config.yml:9]: TIL, thanks

**Review by bruno-garcia — APPROVED** [review]

File: src/platforms/apple/config.yml (hunk 1 above)
Comment (philipphofmann) [reply src/platforms/apple/config.yml]: Whoops you are both right @marandaneto and @bruno-garcia. It should work on server but it doesn't really.

**Pull request merged**